src/renderers/dom/shared/ReactBrowserEventEmitter.js

```diff
@@ -151,6 +152,39 @@ var topEventMapping = {
   topWheel: 'wheel',
```

Comment:
This object is the exact same as [`EventConstants`](https://github.com/facebook/react/blob/master/src/renderers/shared/shared/event/EventConstants.js), sans the values. Has there been any thought on consolidating them into `EventConstants`?